tests/sweep_framework/sweeps/conv2d/short/conv2d_short_sweep.py

```diff
@@ -0,0 +1,450 @@
+            [1, 72, 72, 80, 80, 3, 3, 1, 1, 1, 1, 72, False, 1],
+            [1, 72, 72, 80, 80, 5, 5, 2, 2, 2, 2, 72, False, 1],
+            [1, 720, 720, 14, 14, 3, 3, 1, 1, 1, 1, 6, False, 1],
+            [1, 720, 720, 17, 17, 5, 5, 1, 1, 2, 2, 720, False, 1],
+            [1, 720, 720, 21, 21, 5, 5, 2, 2, 0, 0, 720, False, 1],
+            [1, 720, 720, 28, 28, 3, 3, 2, 2, 1, 1, 6, False, 1],
+            [1, 728, 728, 38, 38, 3, 3, 1, 1, 1, 1, 728, False, 1],
+            [1, 7392, 7392, 24, 24, 3, 3, 2, 2, 1, 1, 28, False, 1],
+            [1, 784, 784, 14, 14, 3, 3, 2, 2, 1, 1, 49, False, 1],
+            [1, 784, 784, 7, 7, 3, 3, 1, 1, 1, 1, 49, False, 1],
+            [1, 8, 8, 112, 112, 3, 3, 1, 1, 1, 1, 8, False, 1],
+            [1, 80, 80, 14, 14, 3, 3, 1, 1, 1, 1, 80, False, 1],
+            [1, 80, 80, 56, 56, 1, 1, 1, 1, 0, 0, 1, False, 1],
+            [1, 80, 80, 56, 56, 3, 3, 1, 1, 1, 1, 1, False, 1],
+            [1, 816, 816, 19, 19, 5, 5, 1, 1, 2, 2, 816, False, 1],
+            [1, 816, 816, 23, 23, 5, 5, 2, 2, 0, 0, 816, False, 1],
+            [1, 88, 88, 28, 28, 3, 3, 1, 1, 1, 1, 88, False, 1],
+            [1, 888, 888, 14, 14, 3, 3, 2, 2, 1, 1, 37, False, 1],
+            [1, 888, 888, 7, 7, 3, 3, 1, 1, 1, 1, 37, False, 1],
+            [1, 896, 896, 14, 14, 3, 3, 1, 1, 1, 1, 7, False, 1],
+            [1, 896, 896, 14, 14, 3, 3, 1, 1, 1, 1, 16, False, 1],
+            [1, 896, 896, 28, 28, 3, 3, 2, 2, 1, 1, 7, False, 1],
+            [1, 896, 896, 28, 28, 3, 3, 2, 2, 1, 1, 16, False, 1],
+            [1, 912, 912, 14, 14, 3, 3, 2, 2, 1, 1, 38, False, 1],
+            [1, 912, 912, 7, 7, 3, 3, 1, 1, 1, 1, 38, False, 1],
+            [1, 92, 92, 14, 14, 3, 3, 1, 1, 1, 1, 92, False, 1],
+            [1, 96, 96, 112, 112, 3, 3, 2, 2, 1, 1, 96, False, 1],
+            [1, 96, 96, 112, 112, 3, 3, 2, 2, 1, 1, 2, False, 1],
+            [1, 96, 96, 113, 113, 3, 3, 2, 2, 0, 0, 96, False, 1],
+            [1, 96, 96, 121, 121, 3, 3, 2, 2, 0, 0, 96, False, 1],
+            [1, 96, 96, 131, 131, 3, 3, 2, 2, 0, 0, 96, False, 1],
+            [1, 96, 96, 28, 28, 5, 5, 2, 2, 2, 2, 96, False, 1],
+            [1, 192, 96, 56, 56, 1, 1, 1, 1, 0, 0, 1, False, 1],
+            [1, 96, 96, 56, 56, 3, 3, 1, 1, 1, 1, 2, False, 1],
+            [1, 96, 96, 56, 56, 1, 1, 1, 1, 0, 0, 1, False, 1],
+            [1, 960, 960, 24, 24, 5, 5, 1, 1, 2, 2, 960, False, 1],
+            [1, 960, 960, 27, 27, 5, 5, 2, 2, 0, 0, 960, False, 1],
+            [1, 960, 960, 3, 3, 1, 5, 1, 1, 0, 2, 960, False, 1],
+            [1, 960, 960, 3, 3, 5, 1, 1, 1, 2, 0, 960, False, 1],
+            [1, 960, 960, 7, 7, 3, 3, 1, 1, 1, 1, 960, False, 1],
+            [1, 960, 960, 7, 7, 5, 5, 1, 1, 2, 2, 960, False, 1],
+        ],
+        "sharding_scheme": [
+            ttnn.TensorMemoryLayout.HEIGHT_SHARDED,
+            ttnn.TensorMemoryLayout.WIDTH_SHARDED,
+            ttnn.TensorMemoryLayout.BLOCK_SHARDED,
```

Comment:
Pavle is working on auto-sharding selection -- this is the PR: https://github.com/tenstorrent/tt-metal/pull/13573
May be wait for this to land in `main` and then you can remove this sharding_scheme parameter and use the auto shard -- this will reduce the #tests by a third.

Reply:
Sure.